lib/plugins/helper/relative_url.js

```diff
@@ -1,29 +1,6 @@
 'use strict';
 
-function relativeUrlHelper(from = '', to = '') {
-  const fromParts = from.split('/');
-  const toParts = to.split('/');
-  const length = Math.min(fromParts.length, toParts.length);
-  let i = 0;
-
-  for (; i < length; i++) {
-    if (fromParts[i] !== toParts[i]) break;
-  }
-
-  let out = toParts.slice(i);
-
-  for (let j = fromParts.length - i - 1; j > 0; j--) {
-    out.unshift('..');
-  }
-
-  const outLength = out.length;
-
-  // If the last 2 elements of `out` is empty strings, replace them with `index.html`.
-  if (outLength > 1 && !out[outLength - 1] && !out[outLength - 2]) {
-    out = out.slice(0, outLength - 2).concat('index.html');
-  }
-
-  return out.join('/').replace(/\/{2,}/g, '/');
-}
-
-module.exports = relativeUrlHelper;
+module.exports = function(from, to) {
+  const relative_url = require('hexo-util').relative_url.bind(this);
+  return relative_url(from, to);
+};
```

Comment:
IMHO, require should be used outside exports to avoid extra overhead, just as I did in #3701 . [Reference](https://stackoverflow.com/questions/31354076/how-to-use-require-inside-global-and-local-function)

```js
const { relative_url } = require('hexo-util');

module.exports = function(from, to) {
  return relative_url.call(this, from, to);
};
```

Reply:
Alright, I also refactored tag plugins.